src/Dapr.Actors.Generators/Helpers/SyntaxFactoryHelpers.cs

```diff
@@ -0,0 +1,159 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Dapr.Actors.Generators.Helpers
+{
+    /// <summary>
+    /// Syntax factory helpers for generating syntax.
+    /// </summary>
+    public static partial class SyntaxFactoryHelpers
```

Comment:
Do these need to be public or can they just be internal?

Reply:
You are right